docs/Atlases.md

```diff
@@ -78,7 +78,7 @@ It was built on anatomical MRI of 24 healthy subjects from which 74 cortical ROI
 It is possible to run the [`t1-volume`](../T1_Volume) and [`pet-volume`](../PET_Volume) <!--and [`dwi-dti`](../DWI_DTI)--> pipelines using a custom parcellation.
 To do so:
 
-- Install Clinica following the [developer instructions](../../Installation/#install-clinica);
+- Install Clinica following the [developer instructions](../Installation/#install-clinica);
```

Comment:
In that link we only ask for installation through `pip install` and not through the source code. Was there another page or shouldn't we add that part in the documentation?

Reply:
My bad the developer instructions are there but hidden by default in an expandable box.

Reply:
See #358